dxr/plugins/rust/__init__.py

```diff
@@ -436,8 +436,13 @@ def process_csv(self, file_name, header_only):
                         print " - 'process_" + line[0] + "' not implemented!"
                         continue
 
-                    if 'file_name' in args and args['file_name'].startswith(self.tree.source_folder):
-                        args['file_name'] = args['file_name'][len(self.tree.source_folder)+1:]
+                    if 'file_name' in args:
+                        file_name_value = os.path.normpath(args['file_name'])
```

Comment:
It does normalization. Here's the source, which probably hasn't changed for 10 years:

```python
def relpath(path, start=curdir):
    """Return a relative version of a path"""

    if not path:
        raise ValueError("no path specified")

    start_list = [x for x in abspath(start).split(sep) if x]
    path_list = [x for x in abspath(path).split(sep) if x]

    # Work out how much of the filepath is shared by start and path.
    i = len(commonprefix([start_list, path_list]))

    rel_list = [pardir] * (len(start_list)-i) + path_list[i:]
    if not rel_list:
        return curdir
    return join(*rel_list)
```

If `start` is not part of `path`, it doesn't error out, but it doesn't return an abspath either: it returns a relpath, like `../../../a/b` for `relpath('/a/b', start='/home/vagrant/src')`. Will that do?